core/src/test/java/com/predic8/membrane/core/IntegrationTestsWithInternet.java

```diff
@@ -32,7 +31,6 @@
         MethodTest.class,
         RegExReplaceInterceptorTest.class,
         LoadBalancingInterceptorTest.class,
```

Comment:
Do you have to include the SeviceIncovationTest class?

Reply:
This was a duplicate test doing the same thing as the other I removed it